coral/src/app/layout/header/ProfileDropdown.tsx

```diff
@@ -34,21 +34,24 @@ const menuItems: MenuItem[] = [
 const LOGOUT_KEY = "logout";
 function ProfileDropdown() {
   const authUser = useAuthContext();
-  const toast = useToast();
+
   function navigateToAngular(path: string) {
     window.location.assign(`${window.origin}${path}`);
   }
 
   function onDropdownClick(actionKey: string | number) {
+    const [toast, dismiss] = useToastContext();
```

Comment:
Was there a reason to move this in the body of the `onDropdownClick` function?

Because in this state, clicking Logout` throws an error, as hooks can only be called in the body of a component:



https://github.com/Aiven-Open/klaw/assets/20607294/8b6bb060-369c-4924-bb90-c59963cac1f3

I think moving it back where it was before (under `const authUser = useAuthContext();`) should fix the issue and still preserve the behaviour we want.